db/migrate/20171110163345_make_subscriber_address_nullable.rb

```diff
@@ -0,0 +1,5 @@
+class MakeSubscriberAddressNullable < ActiveRecord::Migration[5.1]
+  def change
+    change_column_null :subscribers, :address, true
```

Comment:
My only concern with allowing `null` values here is it's possible to end up with strings which are either `null` or `""` which effectively mean the same thing in our system. Perhaps the validation will catch any problems here though.

Reply:
Good point. We should only have one version of empty so it's not confusing.

I've added a test to ensure this can't be an empty string so this column can only be `nil`. We could do it the other way round, Rails has good out-of-the-box support for `allow_nil` that we can take advantage of this way round.

Here's the test: https://github.com/alphagov/email-alert-api/pull/229/files#diff-16d4cbe1eafec712dda5d67ccfcca877R38

What do you think?